src/main/java/com/vfasad/service/UserService.java

```diff
@@ -67,11 +67,12 @@ public void updateUser(User user) {
     }
 
     public List<User> getAdmins() {
-        return userRepository.getByRoleIn("ROLE_ADMIN");
+        return userRepository.getByRoleInOrderByName("ROLE_ADMIN");
     }
 
     public List<User> getManagers() {
-        return userRepository.getByRoleIn("ROLE_SALES", "ROLE_OPERATOR", "ROLE_ADMIN");
+        return userRepository.getByRoleInOrderByName("ROLE_SALES", "ROLE_OPERATOR", "ROLE_ADMIN")
+                .stream().filter(u -> !u.isDeleted()).collect(Collectors.toList());
```

Comment:
It would be better to filter deleted users in query, not in memory. It doesn't really matter in our case, so it's just for future.